src/language/lang_tr.php

```diff
@@ -0,0 +1,659 @@
+  'Saved' =>
+    'Kaydedildi',
+  'All changes have been successfully saved.' =>
+    'Bütün değişiklikler başarıyla kaydedildi.',
+  'Error' =>
+    'Hata',
+  'Sorry your form was not saved. Please correct the all errors and save again.' =>
+    'Formunuz kaydedilmediği için üzgünüz. Lütfen tüm hataları düzeltip yeniden kaydedin.',
+  'cancel_' =>
+    'iptal_',
+  'Are you sure you want to cancel? You have unsaved changes that will be reverted.' =>
+    'İptal etmek istediğinizden emin misiniz? Geri alınacak olan kaydedilmemiş değişiklikleriniz var.',
+  'choose_logo' =>
+    'logo_seç',
+  'captured_' =>
+    'ele_geçirildi_',
+  'flag_owner_' =>
+    'flag_sahibi_',
+  'INACTIVE' =>
+    'ETKİSİZ',
+  'PTS' =>
+    'PUAN',
+  'category' =>
+    'kategori',
+  'capture_' =>
+    'ele_geçirme_',
+  'Insert your answer' =>
+    'Yanıtınızı girin',
+  'Request Hint' =>
+    'İpucu İste',
+  'Submit' =>
+    'Gönder',
+  'hint_' =>
+    'ipucu_',
+  'first_capture' =>
+    'ilk_ele_geçirme',
+  'completed_by' =>
+    'tarafından_tamamlandı',
+  'scoreboard_' =>
+    'skor_tablosu_',
+  'filter_' =>
+    'filtre_',
+  'rank_' =>
+    'derece_',
+  'team_name_' =>
+    'takım_adı_',
+  'quiz_pts_' =>
+    'quiz_puanı_',
+  'flag_pts_' =>
+    'flag_puanı_',
+  'base_pts_' =>
+    'base_puanı_',
+  'total_pts_' =>
+    'toplam_puan_',
+  'team_' =>
+    'takım_',
+  'team_members' =>
+    'takım_üyeleri',
+  'base_pts' =>
+    'base_puanı',
+  'quiz_pts' =>
+    'quiz_puanı',
+  'flag_pts' =>
+    'flag_puanı',
+  'total_pts' =>
+    'toplam_puan',
+  'Tool bars are located on all edges of the gameboard. Tap a category to expand and close each tool bar.' =>
+    'Araç çubukları oyun panosunun tüm kenarlarında bulunmaktadır. Bir araç çubuğunda bulunan kategorileri tıklayarak genişletebilir ve kapatabilirsiniz.',
+  'Tool_Bars' =>
+    'Araç_Çubukları',
+  'Tap the "Game Clock" to keep track of time during gameplay. Don’t let time get the best of you.' =>
+    'Oyun sırasında zamanı takip etmek için "Oyun Saati"ne tıklayın. Zamanın sizi alt etmesine izin vermeyin.',
+  'Game_Clock' =>
+    'Oyun_Saati',
+  'Countries marked with an ' =>
+    'Şu simge ile işaretli ülkeler: ',
+  'are captured by you.' =>
+    'sizin tarafınızdan ele geçirildi.',
+  ' are owned by others.' =>
+    'başkaları tarafınızdan ele geçirildi.',
+  'Captures' =>
+    'Ele Geçirmeler',
+  'Tap Plus[+] to Zoom In. Tap Minus[-] to Zoom Out.' =>
+    'Yakınlaşmak için Artı[+] işaretine tıklayın. Uzaklaşmak için Eksi[-] işaretine tıklayın.',
+  'Click and Drag to move left, right, up and down.' =>
+    'Sola, sağa, yukarı ve aşağı hareket etmek için tıklayıp sürükleyin.',
+  'Zoom' =>
+    'Yakınlık',
+  'Tap Forward Slash [/] to activate computer commands. A list of commands can be found under "Rules".' =>
+    'Bilgisayar komutlarını etkinleştirmek için Bölü[/] simgesine tıklayın. Komut listesi "Kurallar" altında bulunabilir.',
```

Comment:
Suggested Change: 'Bilgisayar komutlarını etkinleştirmek için Eğik Çizgi[/] simgesine tıklayın. Komut listesi "Kurallar" altında bulunabilir.'